controllers/internal/ibmblockcsi/ibmblockcsi.go

```diff
@@ -127,6 +135,13 @@ func (c *IBMBlockCSI) GetCSINodeImage() string {
 	return c.Spec.Node.Repository + ":" + c.Spec.Node.Tag
 }
 
+func (c *IBMBlockCSI) GetCallHomeImage() string {
+	if c.Spec.CallHome.Tag == "" {
+		return c.Spec.CallHome.Repository
+	}
+	return c.Spec.CallHome.Repository + ":" + c.Spec.CallHome.Tag
+}
```

Comment:
could you move this logic to a common `GetComponentImage(repository string, tag string) string` function?

Reply:
done

Reply:
I think it would be nicer to keep the existing functions (without much logic, just wrapping the parameters), what do you think?

Reply:
np, done